src/main/java/org/broadinstitute/hellbender/cmdline/StandardArgumentDefinitions.java

```diff
@@ -92,4 +92,6 @@ private StandardArgumentDefinitions(){}
     public static final String USE_JDK_INFLATER_SHORT_NAME = "jdk-inflater";
     public static final String NIO_MAX_REOPENS_LONG_NAME = "gcs-max-retries";
     public static final String NIO_MAX_REOPENS_SHORT_NAME = "gcs-retries";
+    public static final String NIO_PROJECT_FOR_REQUESTER_PAYS_LONG_NAME = "project-for-requester-pays";
```

Comment:
I think `gcs-project-for-requester-pays` would be better, and more consistent with the existing retry args.

Reply:
OK, done.